tests/Load/config.sh

```
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+
+export $(grep -v '^#' .env | xargs)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)